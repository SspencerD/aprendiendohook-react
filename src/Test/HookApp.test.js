import React from 'react';
import { HookApp } from '../HookApp';
import { shallow } from 'enzyme';



describe('Test en Componente principal HookApp', () => {

    test('Debe enviar el componente dentro del snap', () => {
        
        const wrapper =shallow(<HookApp/>)

        expect(wrapper).toMatchSnapshot();
    });
   
});


