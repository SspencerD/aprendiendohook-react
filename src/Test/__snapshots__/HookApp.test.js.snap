// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test en Componente principal HookApp Debe enviar el componente dentro del snap 1`] = `
<div>
  <h1>
     Hola mundo
  </h1>
</div>
`;

